### ISSUE TITLE
Invalid mode ('rwb')

### ISSUE THREAD

**Issue by pmferreira3** [makinacorpus/django-appypod#2]
Hi,

I am experiencing  the following error:

```
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\django\core\handlers\exception.py", line 39, in inner
    response = get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Python27\lib\site-packages\django\core\handlers\base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "<my project>\views.py", line 67, in generate_document
    response.render()
  File "C:\Python27\lib\site-packages\django\template\response.py", line 109, in render
    self.content = self.rendered_content
  File "C:\Python27\lib\site-packages\django\template\response.py", line 86, in rendered_content
    content = template.render(context, self._request)
  File "C:\Python27\lib\site-packages\djappypod\backend.py", line 54, in render
    with NamedTemporaryFile('rwb', suffix='.odt', delete=False) as f:
  File "C:\Python27\lib\tempfile.py", line 477, in NamedTemporaryFile
    file = _os.fdopen(fd, mode, bufsize)
ValueError: Invalid mode ('rwb')
```
Can you give any hint what is causing the problem?

I am using python 2.7 and django 1.10.3

**Comment by leplatrem** [comment]
Hi!
Thanks for reporting!
It looks like Windows does not support this line...
It's a general Python issue, there might be an easy fix, but I don't know Windows enough :|
Do not hesitate to submit some changes, we'll review, merge and release if necessary!

**Comment by gutard** [comment]
Fixed in version 2.0.0
